examples/fetch_and_cat.rs

```diff
@@ -37,37 +33,49 @@ async fn main() {
         }
     };
 
+    if path.root().cid().is_none() {
+        eprintln!("Unsupported path: ipns resolution is incoming: {}", path);
```

Comment:
assuming this means that `ipns` resolution is an upcoming feature, the following will be a bit more clear:
```suggestion
        eprintln!("Unsupported path: ipns resolution is not supported yet: {}", path);
```